src/AutoMerge/guidelines.jl

```diff
@@ -424,7 +424,8 @@ function meets_repo_url_requirement(pkg::String; registry_head::String)
 end
 
 function _invalid_sequential_version(reason::AbstractString)
-    return false, "Does not meet sequential version number guideline: $reason", :invalid
+    return false, "Does not meet sequential version number guideline: $reason\n \
+    A double check is required to verify this was done on purpose.", :invalid
```

Comment:
What about this?
```suggestion
   If this was intentional, please leave a comment saying so. Otherwise please correct the version number in your package and re-trigger registration.", :invalid
```

Reply:
It's great! I thought it should be brief.

Reply:
Can we mention the Slack channel?

Reply:
I think it already is mentioned somewhere else in the message that gets posted